types/react-redux/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-redux 6.0.4
+// Type definitions for react-redux 6.0.5
```

Comment:
A couple requests:
- Can you remove the patch version from the version? (it should just be 6.0).
- Can you remove the line in the tslint.json in this folder that says "dtheader: false"? That will ensure the header has the right format.

It would be great to get rid of every rule disabling line in the tslint.json, but that might be a lot of work.